packages/template-retail-react-app/app/pages/checkout/partials/shipping-options.jsx

```diff
@@ -138,71 +264,33 @@ export default function ShippingOptions() {
-                                                        <Flex justify="space-between" w="full">
-                                                            <Box>
-                                                                <Text>{opt.name}</Text>
-                                                                <Text
-                                                                    fontSize="sm"
-                                                                    color="gray.600"
-                                                                >
-                                                                    {opt.description}
-                                                                </Text>
-                                                            </Box>
-                                                            <Text fontWeight="bold">
-                                                                <FormattedNumber
-                                                                    value={opt.price}
-                                                                    style="currency"
-                                                                    currency={currency}
-                                                                />
-                                                            </Text>
-                                                        </Flex>
-
-                                                        {opt.shippingPromotions?.map((promo) => {
-                                                            return (
-                                                                <Text
-                                                                    key={promo.promotionId}
-                                                                    fontSize="sm"
-                                                                    color="green.600"
-                                                                >
-                                                                    {promo.calloutMsg}
-                                                                </Text>
-                                                            )
-                                                        })}
-                                                    </Radio>
-                                                )
-                                            )}
-                                        </Stack>
-                                    </RadioGroup>
+                        {/* Dynamically create shipping method options for each shipment */}
+                        {deliveryShipments.map((shipment) => (
+                            <Box key={shipment.shipmentId}>
+                                {hasMultipleDeliveryShipments ? (
+                                    // Multiship: Show both product cards and shipping options
+                                    <ShipmentOptionsWithProducts
```

Comment:
Naming is to verbose. how about `MultiAddressShipmentMethod`?

```suggestion
                                    <MultiAddressShipmentMethod 
```

Reply:
PR: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/3186